src/app.js

```diff
@@ -66,10 +79,13 @@ app.use("*", (req, res, next) => {
 
 
 process.on("uncaughtException", (err) => {
+
   process.exit(0);
 });
 
-process.on("uncaughtExceptionMonitor", (err, origin) => {
+process.on("uncaughtExceptionMonitor", (err) => {
+      console.log(err);
```

Comment:
Codacy has a fix for the issue: [Delete `····`](https://app.codacy.com/gh/armerillo/ql-task/pullRequest?prid=9976636)
```suggestion
  console.log(err);
```